app/notifications/process_notifications.py

```diff
@@ -149,9 +152,18 @@ def send_notification_to_queue(notification, research_mode, queue=None):
         if not queue:
             queue = QueueNames.SEND_EMAIL
         deliver_task = provider_tasks.deliver_email
+    if notification.notification_type == LETTER_TYPE:
+        if not queue:
+            queue = QueueNames.CREATE_LETTERS_PDF
+        deliver_task = create_letters_pdf
 
     try:
         deliver_task.apply_async([str(notification.id)], queue=queue)
+        if notification.notification_type == LETTER_TYPE:
+            update_notification_status_by_reference(
+                notification.reference,
+                NOTIFICATION_DELIVERED,
+            )
```

Comment:
`delivered` is saved for when we've sent to dvla. The letter should stay in `created` (the zip_and_send_to_dvla task only processes letters that are in created)